test/unit_tests/test_interface.py

```diff
@@ -48,7 +59,11 @@ def test_fadump_enabled(self):
         self._check_properties_changed("FadumpEnabled", True)
         self.assertEqual(self._interface.FadumpEnabled, True)
 
-    def test_reserved_memory(self):
+    @patch("builtins.open", MockBuiltinRead(AARCH64_NFO_FIXTURE))
+    @patch("blivet.arch.get_arch", return_value="aarch64")
+    def test_reserved_memory_aarch64(self, _mock_read):
```

Comment:
I don't think you should change this test or test/unit_tests/test_interface.py at all. All tests here in test/unit_tests/test_interface.py are to make sure PropertiesChanged signal will be emitted and then the corresponding callback functions will catch it.